Python/codecs.c

```diff
@@ -50,6 +50,27 @@ int PyCodec_Register(PyObject *search_function)
     return -1;
 }
 
+int
+PyCodec_Unregister(PyObject *search_function)
+{
+    PyInterpreterState *interp = PyInterpreterState_Get();
+    PyObject *codec_search_path = interp->codec_search_path;
+    /* Do nothing if codec_search_path is not created yet or was cleared. */
+    if (codec_search_path == NULL) {
+        return 0;
+    }
+
+    Py_ssize_t n = PyList_Size(codec_search_path);
+    for (Py_ssize_t i = 0; i < n; i++) {
+        PyObject *item = PyList_GetItem(codec_search_path, i);
```

Comment:
Please use the macro `PyList_GET_ITEM()` instead of `PyList_GetItem()`, as `codec_search_path` is guaranteed to be a list.

Reply:
Thanks, done.